src/lib/write-package.ts

```diff
@@ -118,13 +118,15 @@ export const writePackage = async (
   }
 
   console.info('latest repo ', latest.repository);
-
-  const repo = repoToGithub(latest.repository);
+  //tslint:disable-next-line:no-any
+  const repo = repoToGithub((latest as any).permsRepo || latest.repository);
```

Comment:
It would be cool to figure out a test for this field. Otherwise, looking good to me 😄 

Thanks @soldair.

Reply:
I think i'm a little short on bandwidth. we should have one but i'd like to see this merged to unblock the user